src/lava/utils/float2fixed.py

```diff
@@ -1,26 +1,719 @@
 # Copyright (C) 2021-22 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 # See: https://spdx.org/licenses/
-
 """
-This module will contain a tool to perform automated float- to fixed-point
-conversion of of Lava Processes.
-
-Given a set of processes and data, the tool will perform the following steps:
-
-1. Find fixed-point ProcessModels of every floating-point ProcessModel if
-available and throw an error otherwise.
-2. Run the floating-point model and collect activation statistics.
-3. Given activation statistic and type/precision/dynamic range specifications
-of fixed-point models, perform input parameter conversion for fixed-point model.
-4. Replace floating-point ProcessModels by corresponding fixed-point
-ProcessModels
-5. Initialize fixed-point ProcessModels
-
-ProcessModel extraction might be a shared function with compiler.
-
-This will likely lead to some performance degradation of the application so
-perhaps some iterative optimization may have to be included within this
-parameter conversion process. Reuse other libraries for this purpose where
-possible.
- """
+This module containx a tool to perform automated float- to fixed-point
+conversion of Lava Processes.
+"""
+from lava.magma.core.run_configs import RunConfig
+from lava.magma.core.run_conditions import RunSteps
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.compiler.compiler_graphs import (
+    find_processes,
+    ProcGroupDiGraphs)
+from lava.magma.core.process.variable import Var
+from lava.magma.core.process.ports.ports import AbstractSrcPort
+from lava.proc.monitor.process import Monitor
+from lava.magma.core.process.ports.ports import OutPort
+
+import numpy as np
+import inspect
+
+
+class Float2FixedConverter:
+
+    def __init__(self):
+        self.floating_pt_rcfg = None
```

Comment:
write run_cfg. Took me a few seconds to figure out what this abbreviation means, and it's not worth that hassle just to save three characters.

Reply:
Good point, addressed in 4a8c69297ec83114efe9491991d0328736556207.